### PR TITLE
[stable/slackin] release 0.0.1

### PR DESCRIPTION
**Pull request by sameersbn**
[Slackin](https://github.com/rauchg/slackin) is a small server for sending out slack invites.

### CODE REVIEW

File: stable/slackin/.helmignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.git
```
Comment (lachie83) [inline stable/slackin/.helmignore]: please update with helm create generated helmignore

File: stable/slackin/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 24 -}}
```
Comment (lachie83) [inline stable/slackin/templates/_helpers.tpl]: Might be worth updating to the following -- https://github.com/kubernetes/charts/blob/86a5197973b9dcc5593bb7dd26b4ca476e3509e4/stable/prometheus/templates/_helpers.tpl#L24

File: stable/slackin/templates/_helpers.tpl [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 24 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 24 -}}
```
Comment (lachie83) [inline stable/slackin/templates/_helpers.tpl]: Same as above

File: stable/slackin/values.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+image:
+  name: chk1/slackin
+  digest: sha256:476ae1beb981449c1b61794bb9ec52beeff1c312ff54fbcd3010946b991975f8
```
Comment (lachie83) [inline stable/slackin/values.yaml:3]: Why this as apposed to image tag?

**Comment by lachie83** [comment]
@k8s-bot ok to test

File: stable/slackin/values.yaml (hunk 4 above)
Comment (sameersbn) [reply stable/slackin/values.yaml:3]: The `chk1/slackin` image does not tag releases.
If this is a problem, then I can create and maintain a slackin image myself.

**Comment by sameersbn** [comment]
suggested changes applied.

**Comment by viglesiasce** [comment]
@sameersbn I don't have a slack org to test with whats the easiest way to test this end to end? 

Also should the hostname be unset without further config? Will it actually work with that default hostname?

**Comment by viglesiasce** [comment]
Closing as stale.

**Comment by luxas** [comment]
This would be nice to have though :)

**Pull request closed without merging**